util/omrutil/j9memclr.cpp

```diff
@@ -28,7 +28,10 @@
 
 #if (defined(LINUX) && defined(S390))
 /* _j9Z10Zero() is defined in j9memclrz10_31.s and j9memclrz10_64.s */
-extern void _j9Z10Zero(void *ptr, uintptr_t length);
+extern "C" void _j9Z10Zero(void *ptr, uintptr_t length);
+#endif
+#if (J9ZOS390)
+extern "C" void J9ZERZ10(void *ptr, uintptr_t length);
 #endif
```

Comment:
style nit, should be more like:
```c++
/* _j9Z10Zero() is defined in j9memclrz10_31.s and j9memclrz10_64.s */
#if defined(LINUX) && defined(S390)
extern "C" void _j9Z10Zero(void *ptr, uintptr_t length);
#elif defined(J9ZOS390)
extern "C" void J9ZERZ10(void *ptr, uintptr_t length);
#endif
```

Reply:
At least, that's how I would write it, fine if you think it's technically two different conditions. Either way, we're still missing a `defined(...)` around `J9ZOS390`.